man/dracut.8.asc

```diff
@@ -18,13 +18,8 @@ DESCRIPTION
 
 Create an initramfs <image> for the kernel with the version <kernel version>.
 If <kernel version> is omitted, then the version of the actual running
-kernel is used. If <image> is omitted or empty, depending on bootloader
-specification, the default location can be
-_/efi/<machine-id>/<kernel-version>/initrd_,
-_/boot/<machine-id>/<kernel-version>/initrd_,
-_/boot/efi/<machine-id>/<kernel-version>/initrd_,
-_/lib/modules/<kernel-version>/initrd_ or
-_/boot/initramfs-<kernel-version>.img_.
+kernel is used. If <image> is omitted or empty, the default location will be
+determined by the local configuration or Linux distribution policy.
```

Comment:
This is not actually true. We are only adding customization for the file name, not the paths. So this is a bit excessive.

Reply:
I agree with the change- documentation is aimed at users; who either use the default, or can override it explicitly.